src/server/pps/server/api_server.go

```diff
@@ -3276,18 +3245,17 @@ func (a *apiServer) ActivateAuth(ctx context.Context, req *pps.ActivateAuthReque
 	defer func(start time.Time) { a.Log(req, resp, retErr, time.Since(start)) }(time.Now())
 	pachClient := a.env.GetPachClient(ctx)
 
+	// Set the permissions on the spec repo so anyone can read it
+	if err := pachClient.ModifyRepoRoleBinding(ppsconsts.SpecRepo, auth.AllClusterUsersSubject, []string{auth.RepoReaderRole}); err != nil {
```

Comment:
Removed the special casing for the spec repo, it now has normal role bindings like every other repo.

Reply:
Maybe I'm missing something - pipeline workers have the pipeline token, no? The alternative would be that it was possible for unauthenticated users to read all spec commits in 1.12.x which doesn't seem right?